Modules/_sqlite/connection.c

```diff
@@ -1983,7 +1983,7 @@ Returns iterator to the dump of the database in an SQL text format.
 [clinic start generated code]*/
 
 static PyObject *
-pysqlite_connection_iterdump_impl(pysqlite_Connection *self)
+pysqlite_connection_iterdump_impl(pysqlite_Connection *self, const char *filter)
```

Comment:
You'll want Argument Clinic to generate the `filter` parameter for you:

```diff
diff --git a/Modules/_sqlite/connection.c b/Modules/_sqlite/connection.c
index 2d5a7a10ee..9bb6a0a60e 100644
--- a/Modules/_sqlite/connection.c
+++ b/Modules/_sqlite/connection.c
@@ -1979,6 +1979,9 @@ pysqlite_connection_interrupt_impl(pysqlite_Connection *self)
 /*[clinic input]
 _sqlite3.Connection.iterdump as pysqlite_connection_iterdump
 
+    *
+    filter: object = None
+
 Returns iterator to the dump of the database in an SQL text format.
 [clinic start generated code]*/
 
```

After this change, you can do `make clinic`, which will regenerate `Modules/_sqlite/clinic/connection.c.h` correctly for you.

Reply:
Many thanks! Now I understand the workflow.